pkgs/development/libraries/totem-pl-parser/default.nix

```diff
@@ -25,11 +25,15 @@ stdenv.mkDerivation rec {
     "-Dintrospection=false"
   ];
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    sed -i "s/,--version-script,.*$/'/" plparse/meson.build
```

Comment:
Probably would be fixed by https://gitlab.gnome.org/GNOME/totem-pl-parser/-/merge_requests/46.

Reply:
Indeed, but since the upstream PR is not rebased, we can't simply do a fetchpatch. And the version-script sed hack is already in 6 packages in nixpkgs, so I think it should be OK to do that.

Reply:
Or I can just remove the dependency by disabling playlist in tracker-miners, if that's preferable.

Reply:
I think this is fine but we should add a link to the MR so that we can check if it is merged when updating.

Reply:
Could you please put the comment just above the `sed` line so that if we add more replacements, it is clear what it belongs to?